_data/tool_and_resource_list.yml

```diff
@@ -1,3 +1,16 @@
+# List of tools and resources
+#
+# When referencing tools and resources use the following notion e.g. on Task pages -  {% tool "github" %} and not a markdown link
+# for this to work the tool has to be listed here.
+#
+# - description: A short description of the purpose of the tool or resource
+#   id: This is the id by which the tool will be references on other pages e.g. for GitHub the id is github so we use {% tool "github"%} to refer to it
+#   name: This is the name of the tool or resource email: email adress
+#   registry: This is a list of registry:search-string combinations 
+#       tess: <name>  <-- this will search the tess training registry using the term <name> - i.e. the tool name and return associated results - in this case around training on this tool
+#   url: the homepage of the tool/resource
+#   
```

Comment:
```suggestion
#   qualityDimension: Research Software Quality Dimension identifier (see dimensions.yaml in this folder) of the quality dimension associated with this tool. More than one dimension may apply (in a list)
#   qualityIndicator:  Research Software Quality indicator (see indicators.yaml in this folder) associated with this tool. More than one quality indicator may be declared per tool (in a list)
```

Reply:
I don't like camel case  - and for it to be CamelCase it would be QualityDimension - I think it's much more readable if we use lower_case_with_underscores so quality_dimension and quality_indicator

Reply:
schema.org uses camelCase notation, and CodeMeta as well (https://codemeta.github.io/terms/), plus half of semantic web vocabularies. The schema we are proposing is aligned with them. Adding a different tag for rsqkit makes things more difficult to maintain in my opinion. 
qualityDimension is correct camel case notation https://en.wikipedia.org/wiki/Camel_case